apps/air-discount-scheme/backend/src/environments/environment.ts

```diff
@@ -12,6 +12,18 @@ const devConfig = {
     url: process.env.NATIONAL_REGISTRY_URL,
     username: process.env.NATIONAL_REGISTRY_USERNAME,
     password: process.env.NATIONAL_REGISTRY_PASSWORD,
+    authMiddlewareOptions: {
+      forwardUserInfo: true,
+      tokenExchangeOptions: {
+        issuer: process.env.IDENTITY_SERVER_DOMAIN
+          ? `https://${process.env.IDENTITY_SERVER_DOMAIN}`
```

Comment:
Why doesn't the `IDENTITY_SERVER_DOMAIN` include the `https://`  as well?

Reply:
Im using the same IDSDomain secret everywhere. So I excluded the https:// because Nextauth wants the domain without https in one place.
Adding https to the secret makes Nextauth contact IDS with https://https://identy......

But I might refactor these secrets completely.